Create organization/philanthropy.md (#786)
publiccodenet/about: organization/philanthropy.md

---
type: resource
explains: our policy on philanthropy
---

# Philanthropy policy

> The Foundation for Public Code is currently wholly supported by philanthropists who share our commitment to [our mission](https://about.publiccode.net/organization/mission.html). All donations are unearmarked and go into our general operating budget, in compliance with our status as a Dutch [public benefit organization](https://www.belastingdienst.nl/wps/wcm/connect/bldcontenten/belastingdienst/business/business-public-benefit-organisations/public_benefit_organisations/public_benefit_organisations) (ANBI).

Reviewer: Perhaps we can add in there that it's wholly __financially__ supported. In addition to this it might be good to say that not earmarking is also to prevent influence.

Author: Done!

> The Foundation for Public Code is currently wholly supported by philanthropists who share our commitment to [our mission](https://about.publiccode.net/organization/mission.html). All donations are unearmarked and go into our general operating budget, in compliance with our status as a Dutch [public benefit organization](https://www.belastingdienst.nl/wps/wcm/connect/bldcontenten/belastingdienst/business/business-public-benefit-organisations/public_benefit_organisations/public_benefit_organisations) (ANBI).

Reviewer: I suggest a return and putting the `ANBI` in the link as it's part of the same entity:

```suggestion
The Foundation for Public Code is currently wholly supported by philanthropists who share our commitment to [our mission](https://about.publiccode.net/organization/mission.html).
All donations are unearmarked and go into our general operating budget, in compliance with our status as a Dutch [public benefit organization (ANBI)](https://www.belastingdienst.nl/wps/wcm/connect/bldcontenten/belastingdienst/business/business-public-benefit-organisations/public_benefit_organisations/public_benefit_organisations).
```

> We will be philanthropically funded until we have enough members paying dues to support a sustainable blended financial model.

Reviewer: The goal here is not to have a blended model, it is to make sure all sustainable operations can be payed for with public means to reduce the influence of fundraising activities. At that point philantropy becomes and engine for growth without it being able to influence our existing responsibilities.

Author: We discussed this at great length. We agree that that's the goal, but thought the financial model page was a better place for this information. The "blended model" was a short way of signalling that we expect to need both member dues and philanthropy to operatate for the foreseeable future. 
Do you think it's important to put this future goal on our philanthropy policy page?

Author: I created a separate issue to update the financial model: #787

Reviewer: I think the main goal of this page would be to indicate that we don't let our fundraising impact how we try to achieve our mission, not through influence, having donors add requirements to their donation, or availability, changing our practices to get more money.

To me the notion of a 'blended' model signals that we plan on becoming a fundraising NGO, where we're trying to become public infrastructure.

Perhaps something like 'We will be philantropically funded to support growth and a larger impact while we try the operational core of delivering on our responsibilities from our members, guaranteeing the public purpose.' or something less abstract ;)

Reviewer: So, to answer the question, yes, I think it's important, but the details should definetely go into the financial model :)

Author: This was how I defined it in an early version of the case for support:

The Foundation for Public Code is primarily member-funded, with additional philanthropic support. Core stewardship operations and member relations are covered by membership dues to make sure we’re sustainable in the long term.

We welcome/embrace philanthropy for:
+ awareness raising
+ecosystem growth
+ in-depth education
+ toolset improvement

This lets us try out new ideas, expand our service offering and reach potential new members and partners without compromising our core activities.

Author: @bvhme Check now?

[See our financial model](https://about.publiccode.net/organization/financial-model.html) for more information about our long term financial plans.

## Working together with philanthropists

> We value the experience, expertise and connections of the philanthropists who've chosen to support us. Our donors have helped us scale, grow internationally and communicate for impact.

Reviewer: I suggest a return here as well.

```suggestion
We value the experience, expertise and connections of the philanthropists who've chosen to support us.
Our donors have helped us scale, grow internationally and communicate for impact.
```

Philanthropists are not involved in our prioritization of activities or day to day operations.

[See our governance model](https://about.publiccode.net/organization/governance-model.html) for more information about how the Foundation is governed.

## Donors

> To date, we've received donations from:

Reviewer: Great! Do we also want to disclose amounts? Or do that in the financial reporting when that comes?

Author: We thought the annual report, and possibly press releases, was a great place for this.

* Arnout Schuijff
* Stewart Butterfield
* Josh Elkes

Arnout is one of our co-founders and currently also sits on the board of directors.

## Become a donor

> See [our case for support] or [become a donor](https://about.publiccode.net/CONTRIBUTING.html#institutional-giving-and-philanthropy).

Reviewer 2: ```suggestion
If you're interested in [becoming a donor](https://about.publiccode.net/CONTRIBUTING.html#institutional-giving-and-philanthropy), please read [our case for support] and email us at giving@publiccode.net.
```

Reviewer 2: Adding the giving email address, in case a donor finds his way here.

In the future, we aim for the Foundation for Public Code to be primarily member-funded, with additional philanthropic support. Membership dues should cover core stewardship operations and member relations to make sure we’re sustainable in the long term. We hope to use philanthropic gifts for awareness raising, ecosystem growth, training and toolset improvement.

[See our financial model](https://about.publiccode.net/organization/financial-model.html) for more information about our long term financial plans.

## Working together with philanthropists

We value the experience, expertise and connections of the philanthropists who've chosen to support us.

Our donors have helped us scale, grow internationally and communicate for impact.

Philanthropists are not involved in our prioritization of activities or day to day operations.

[See our governance model](https://about.publiccode.net/organization/governance-model.html) for more information about how the Foundation is governed.

## Donors

We've received donations from:

* Arnout Schuijff
* Josh Elkes
* Stewart Butterfield

Arnout is one of our co-founders and currently also sits on the board of directors.

> ## Support the Foundation for Public Code

Reviewer: Can I suggest

```suggestion
## How to donate
```
